projects/demo/src/app/app.component.ts

```diff
@@ -41,4 +41,12 @@ export class AppComponent {
     loadTextEditIconSet();
     loadTravelIconSet();
   }
+
+  onUseShimChange(e) {
```

Comment:
Playing around with my demo url, this is doing a very bad job of keeping the body attribute in sync with the dropdown. 

I welcome suggestions for a more angular way of doing this.

Reply:
I am looking into this. A directive might be the way to go here.

Reply:
Removing the `*clrIfOpen` directive from the `clr-dropdown-menu` element makes it stop recreating the menu every time so the theme select value doesn’t revert back to "None" when reopening the menu.

Reply:
Updated, thank you!